src/Blueprint/Blueprint/Cmdlets/BlueprintDefinition/ExportAzureRMBlueprint.cs

```diff
@@ -0,0 +1,83 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Blueprint.Models;
+using System.IO;
+using System.Management.Automation;
+using Microsoft.Azure.PowerShell.Cmdlets.Blueprint.Properties;
+using static Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants;
+
+namespace Microsoft.Azure.Commands.Blueprint.Cmdlets
+{
+    [Cmdlet("Export", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "BlueprintWithArtifact", SupportsShouldProcess = true, DefaultParameterSetName =
+         ParameterSetNames.ExportBlueprintParameterSet), OutputType(typeof(string))]
+    public class ExportAzureRmBlueprint : BlueprintDefinitionCmdletBase
+    {
+        #region Parameters
+        [Parameter(ParameterSetName = ParameterSetNames.ExportBlueprintParameterSet, Mandatory = true, ValueFromPipeline = true, HelpMessage = ParameterHelpMessages.ExportBlueprintObject)]
+        [ValidateNotNullOrEmpty]
+        public PSBlueprintBase Blueprint { get; set; }
```

Comment:
@filizt same comment about adding a parameter set for `-BlueprintName`

Reply:
Resolving this with the same reasoning.